src/character.cpp

```diff
@@ -507,6 +507,21 @@ void Character::randomize_height()
     init_height = clamp( x, 145, 200 );
 }
 
+const item *Character::get_wielded_weapon() const
```

Comment:
As weapon is always a valid object there is no need to return a pointer, it can be a reference to avoid using raw pointers.